libclamav/xlm_extract.c

```diff
@@ -5005,10 +5007,14 @@ cl_error_t cli_extract_xlm_macros_and_images(const char *dir, cli_ctx *ctx, char
          * A drawing group was extracted, now we need to find all the images inside.
          * If we fail to extract images, that's fine.
          */
-        (void)cli_extract_images_from_drawing_group(drawinggroup, drawinggroup_len, ctx);
+        ret = cli_extract_images_from_drawing_group(drawinggroup, drawinggroup_len, ctx);
```

Comment:
cli_extract_images_from_drawing_group can also return 'CL_EARG'.  Should we check for that error as well?

Reply:
I don't think i want to.   The can of worms there is that `cli_extract_images_from_drawing_group()`'s default return status code is `CL_EARG` and an oversight in that function has it returning the default error for a bunch of parsing-related (non-arg-related) failures. 

```c I don't want further parsing of the XLS document to be skipped because it failed to extract/scan an image from the drawing group.